src/Microsoft.Build.Sql.Templates/README.md

```diff
@@ -0,0 +1,43 @@
+# Microsoft.Build.Sql.Templates
+
+
+## Using the templates
+
+
+Creating a new project "AdventureWorks":
+
+```bash
+dotnet new sqlproj -n "AdventureWorks"
+```
+
+Displaying help information for the SQL project template:
+
+```bash
+dotnet new sqlproj -h
+```
+
+
+Creating a new project "AdventureWorksLT" for Azure SQL Database:
+
+```bash
+dotnet new sqlproj -n "AdventureWorksLT" -t "SqlAzureV12"
```

Comment:
is `-t` short for `--targetplatform`?

Reply:
it's currently short for `--targetPlatform` but I'm slowly realizing that convention suggests it should be short for `--target-platform` or `--targetplatform`.  Do folks have a preference on hyphen or no?

Reply:
most of the other template options [here](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new-sdk-templates) use hypens, so I think it makes sense to go with hyphen here